compiler/src/test/scala/dx/compiler/CompilerTest.scala

```diff
@@ -753,7 +757,8 @@ class CompilerTest extends AnyFlatSpec with Matchers with BeforeAndAfterAll {
     y.default shouldBe Some(IOParameterValueNumber(5))
   }
 
-  it should "dependency report should be added to WF description" in {
+  // TODO: un-ignore when files are fixed
```

Comment:
I fixed the files that got deleted and updated the test awhile ago. This can be unignored now.